src/kakarot/instructions/environmental_information.cairo

```diff
@@ -397,6 +398,88 @@ namespace EnvironmentalInformation {
         return ctx;
     }
 
+    // @notice EXTCODECOPY operation
+    // @dev Copy an account's code to memory
+    // @custom:since Frontier
+    // @custom:group Environmental Information
+    // @custom:gas 100
+    // TODO: double check correctness of docstring
+    // @custom:stack_consumed_elements 4
+    // @custom:stack_produced_elements 0
+    // @param ctx The pointer to the execution context
+    // @return The pointer to the updated execution context.    
+    func exec_extcodecopy{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr,
+        bitwise_ptr: BitwiseBuiltin*,
+    }(ctx: model.ExecutionContext*) -> model.ExecutionContext* {
+        alloc_locals;
+
+        let stack = ctx.stack;
+
+        // Stack input:     
+        // 0 - address: 20-byte address of the contract to query. 
+        // 1 - dest_offset: byte offset in the memory where the result will be copied.
+        // 2 - offset: byte offset in the code to copy.
+        // 3 - size: byte size to copy.
+        let (stack, popped) = Stack.pop_n(self=stack, n=4);
+        let address_ = popped[0];
+        let dest_offset = popped[1];
+        let offset = popped[2];
+        let size = popped[3];
+      
+        let address = Helpers.uint256_to_felt(address_);
+
+        // Get the starknet address from the given evm address
+        let (registry_address_) = registry_address.read();
+
+        let (starknet_contract_address) = IRegistry.get_starknet_contract_address(
+            contract_address=registry_address_, evm_contract_address=address
+        );
+
+        // handle case where there is no eth -> stark address mapping
+        if (starknet_contract_address == 0) {
```

Comment:
Does reading a uninitialized storage value in Starknet return a 0?

Reply:
I believe so, in practice I get this result and also it is mentioned here: https://hackmd.io/@RoboTeddy/BJZFu56wF#Ambiguity-between-uninitialized-memory-and-the-value-0